commercetools-models/src/test/java/io/sphere/sdk/carts/ExternalPricesIntegrationTest.java

```diff
@@ -0,0 +1,248 @@
+package io.sphere.sdk.carts;
+
+import io.sphere.sdk.carts.commands.CartUpdateCommand;
+import io.sphere.sdk.carts.commands.updateactions.AddLineItem;
+import io.sphere.sdk.carts.commands.updateactions.ChangeLineItemQuantity;
+import io.sphere.sdk.carts.commands.updateactions.RemoveLineItem;
+import io.sphere.sdk.carts.commands.updateactions.SetLineItemPrice;
+import io.sphere.sdk.test.IntegrationTest;
+import org.javamoney.moneta.Money;
+import org.junit.Test;
+
+import javax.money.MonetaryAmount;
+
+import static io.sphere.sdk.carts.CartFixtures.withCartWithLineItems;
+import static io.sphere.sdk.carts.CartFixtures.withEmptyCartAndProduct;
+import static io.sphere.sdk.carts.LineItemPriceMode.EXTERNAL_PRICE;
+import static io.sphere.sdk.carts.LineItemPriceMode.EXTERNAL_TOTAL;
+import static io.sphere.sdk.carts.LineItemPriceMode.PLATFORM;
+import static io.sphere.sdk.products.ProductFixtures.withProduct;
+import static io.sphere.sdk.test.SphereTestUtils.*;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ExternalPricesIntegrationTest extends IntegrationTest {
+
+    @Test
+    public void createCartWithALineItemWithAnExternalPrice() {
+        withProduct(client(), product -> {
+            final int quantity = 5;
+            final int variantId = 1;
+            final Money externalPrice = Money.from(EURO_12);
```

Comment:
After applying the changes from above, you can replace `Money.from(...)` with `EURO_12` here.